### PR TITLE
feat: update dependencies to use proof aggregation branches

### PR DESCRIPTION
**Pull request by cryptonemo**
This is a WIP getting the dependencies required for proof aggregation hooked in. The API is not wired through yet, but will be added here.

### CODE REVIEW

**Comment by cryptonemo** [comment]
The API has been wired in now and end to end aggregation testing works via the `--ignored` rust tests.

File: rust/src/proofs/api.rs [hunk 1]
```diff
@@ -402,6 +404,166 @@ pub unsafe extern "C" fn fil_seal_commit_phase2(
...
+            }
+        }
+
+        info!("seal_commit_phase2_for_aggregation: finish");
+
+        raw_ptr(response)
+    })
+}
+
+pub unsafe extern "C" fn fil_aggregate_seal_proofs(
```
Comment (magik6k) [inline rust/src/proofs/api.rs:473]: It seems like the Go method for this didn't get generated

Comment (cryptonemo) [reply rust/src/proofs/api.rs:473]: Ok, I've got most of the re-factor working locally, but am not sure how it affects your commit.  I will likely open a new draft PR on a different branch and we can resolve which is better later.

**Comment by cryptonemo** [comment]
This PR is outdated now compared to this alternative: https://github.com/filecoin-project/filecoin-ffi/pull/169

**Pull request closed without merging**